src/main/kotlin/ratpack/example/kotlin/MyModule.kt

```diff
@@ -8,18 +8,21 @@ import ratpack.handling.HandlerDecorator
  * An example Guice module.
  */
 class MyModule : AbstractModule() {
-
-  /**
-   * Adds a service impl to the application, and registers a decorator so that all requests are logged.
-   * Registered implementations of {@link ratpack.handling.HandlerDecorator} are able to decorate the application handler.
-   *
-   * @see MyHandler
-   */
-  protected override fun configure() {
-    bind(MyService::class.java).to(MyServiceImpl::class.java)
-    bind(MyHandler::class.java)
-    Multibinder.newSetBinder(binder(), HandlerDecorator::class.java)
-      .addBinding()
-      .toInstance(HandlerDecorator.prepend(LoggingHandler()))
-  }
+    /**
+     * Adds a service impl to the application, and registers a decorator so that all requests are logged.
+     * Registered implementations of {@link ratpack.handling.HandlerDecorator} are able to decorate the
+     * application handler.
+     *
+     * @see MyHandler
+     */
+    override fun configure() {
+        bind(MyService::class.java).to(MyServiceImpl::class.java)
+        bind(MyHandler::class.java)
+        Multibinder.newSetBinder(binder(), HandlerDecorator::class.java)
+            .addBinding()
+            .toInstance(HandlerDecorator.prepend {
+                println("Received: ${it.request.uri}")
```

Comment:
I would keep `LoggingHandler `, it's demonstrating a handler implicitly set up by a module